docs/tutorials/piaf.rst

```diff
@@ -82,6 +82,27 @@ the latter element could be accessed directly as follows:
           use either raw strings (using the `r` prefix, as in the example
           above) or escape each backslash as `\\\\\\\\`.
 
+.. _finding_attributes:
+
+************************************
+Finding attributes based on full path
+************************************
+
+To get the direct attribute based on the entire element/attributes path
+you can use the :any:`PIAFElement.attributes` method. You can provide a single string or list of strings with
```

Comment:
You defined the method on `PIAFDatabase`, not on `PIAFElement`, could you please correct that?

Reply:
kept it at PIAFDatabase to keep the parallel with PI.py Search()

Reply:
Sorry for the confusion, but I meant that this line in the documentation mentions it is an attribute on a `PIAFElement`. That could cause confusion to others, even though the code sample below mentions the correct usage.